routes/web.php

```diff
@@ -33,4 +35,5 @@
     Route::get('/attribute/edit', [AttributesController::class, 'editAttribute'])->name('admin.attributes.edit');
     Route::get('/showusers', [UsersController::class, 'showUsers'])->name('showusers');
     Route::get('/admins/register', [AdminsController::class, 'registerAdmin'])->name('admins.register');
+    Route::get('/areas/edit',[AreasController::class,'editRegisteredAreas'])->name('admin.areas.edit');
```

Comment:
Please see comments on PR#17 first regarding the redundant admin route :) 

`->name('admin.areas.edit');` will look something like `admin.admin.areas.edit` ❌

Reply:
@KobeSergio 
Thank you. I've changed it to ->name('areas.edit');.
Is it OK with it?

Reply:
Yes, its okay now :)